validation/src/main/resources/application.conf

```diff
@@ -158,6 +158,11 @@ hmda.validation.macro {
   Q065 {
     hoepaLoanLimit = 200
   }
+  Q071 {
+    currentYearThreshold = 50
```

Comment:
In reference to Q071, how are these values used in the calculations?  When reading the ticket, I see a threshold of 2500, a current year ratio of 30%, and a 10% difference between the two years.